physionet-django/user/test_integration.py

```diff
@@ -211,7 +212,7 @@ def test_credential_application(self):
             'application-reference_email': 'root@example.com',
             'application-reference_organization': 'MIT',
             'application-reference_title': 'Administrator',
-            'application-research_summary': 'Effects of asdfghjk on zxcvbnm',
+            'application-research_summary': Faker().text(nb_words=100, ext_word_list=None),
```

Comment:
Do we need faker here? Perhaps just provide a sample sentence (i.e. the lengthen the current example).

Reply:
Adding 100 words in length in string would be too long (pass the line limit for PEP), so I decided to use the faker library that can generate random text, to generate a summary of at least 100 words long.

Reply:
Using the string multiplication would also work #e255db1
` data = {
    'application-research_summary': 'Effects of asdfghjk on zxcvbnm.' * 10 `

Reply:
> Adding 100 words in length in string would be too long (pass the line limit for PEP)

The style guidelines shouldn't stop you from writing the code that you want to write! If you want a long string, then put it on multiple lines.